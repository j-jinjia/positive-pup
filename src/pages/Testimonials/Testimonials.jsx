import Layout from "../../components/Layout/Layout";

const Testimonials = () => {
  return (
    <Layout>
      <h1>Testimonials</h1>
    </Layout>
  );
};

export default Testimonials;
