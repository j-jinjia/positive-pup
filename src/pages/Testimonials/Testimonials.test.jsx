import { customRender } from "../../utils/testUtils";
import Testimonials from "./Testimonials";

it("should render the testimonials page", () => {
  const { container } = customRender(<Testimonials />);

  expect(container).toMatchSnapshot();
});
