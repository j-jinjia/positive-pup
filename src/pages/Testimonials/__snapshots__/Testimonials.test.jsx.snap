// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the testimonials page 1`] = `
<div>
  <div
    class="layout"
  >
    <div
      class="navbar"
    >
      <div
        class="navbar__container"
      >
        <a
          href="/"
        >
          <img
            alt="The Positive Pup logo"
            class="navbar__container-logo"
            src="navbar-logo.png"
          />
        </a>
        <nav
          class="navbar__nav"
        >
          <a
            class="navbar__nav-item"
            href="/about"
          >
            About
          </a>
          <button
            class="navbar__nav-item navbar__nav-item--button"
          >
            Services
            <img
              class="navbar__nav-item-arrow"
              src="dropdown-arrow.svg"
            />
          </button>
          <a
            class="navbar__nav-item"
            href="/testimonials"
          >
            Testimonials
          </a>
          <a
            class="navbar__nav-item"
            href="/gallery"
          >
            Gallery
          </a>
          <span
            class="navbar__nav-item navbar__nav-item--phone"
          >
            <img
              alt=""
              class="navbar__nav-item-button"
              src="phone.svg"
            />
            <a
              class="navbar__nav-item"
              href="/get-in-touch"
            >
              Get in Touch
            </a>
          </span>
          <div
            class="button button--primary button--link"
          >
            <a
              href="/book-now"
            >
              Book Now
            </a>
          </div>
        </nav>
      </div>
    </div>
    <h1>
      Testimonials
    </h1>
    <div
      class="footer"
    >
      <div
        class="footer__image-container"
      >
        <img
          alt="logo"
          class="footer__logo"
          src="footer-logo.svg"
        />
      </div>
      <ul
        class="footer__links-container"
      >
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/services"
          >
            Services
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/gallery"
          >
            Gallery
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--desktop"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            Terms & Conditions
          </a>
        </li>
        <li
          class="footer__list-item footer__list-item--mobile"
        >
          <a
            class="footer__link"
            href="/terms-and-conditions"
          >
            T&C's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/faqs"
          >
            FAQ's
          </a>
        </li>
        <li
          class="footer__list-item"
        >
          <a
            class="footer__link"
            href="/get-in-touch"
          >
            Contact
          </a>
        </li>
      </ul>
      <hr
        class="footer__line-break"
      />
      <div
        class="footer__secondary-links-container"
      >
        <div
          class="footer__copyright-text"
        >
          Â© 2022 The Positive Pup. All rights reserved.
        </div>
        <div
          class="footer__social-links"
        >
          <a
            href="https://www.facebook.com/thepositivepupdogtraining"
            target="blank"
          >
            <img
              alt="link to facebook"
              src="facebook-logo.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/the_positivepup/"
            target="blank"
          >
            <img
              alt="link to instagram"
              src="instagram-logo.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
